htmd/molecule/molecule.py

```diff
@@ -1279,6 +1232,15 @@ def write(self, filename, sel=None, type=None, **kwargs):
             raise IOError('Molecule cannot write files with "{}" extension yet. If you need such support please notify '
                           'us on the github htmd issue tracker.'.format(ext))
 
+    def _emptyTopo(self, numAtoms):
+        for field in Molecule._atom_fields:
+            self.__dict__[field] = self._empty(numAtoms, field)
+
+    def _emptyTraj(self, numAtoms):
```

Comment:
shouldn't it initialize all `Molecule._traj_fields`?

Reply:
We cannot initialize `'fileloc', 'step', 'time'` fields

Reply:
because they are not np.arrays